plugins/woocommerce-admin/docs/examples/extensions/examples.config.js

```diff
@@ -103,6 +103,12 @@ const webpackConfig = {
 		} ),
 		new WooCommerceDependencyExtractionWebpackPlugin(),
 	],
+	watchOptions: {
+		ignored: [
+			path.resolve( __dirname, `../../../../${ extension }/` ),
```

Comment:
When I change any file, say `plugins/add-task/js/index.js` it doesn't rebuild. Could this `ignore` be causing that?

Reply:
Nope, this is unrelated but I've just changed it to ignore `dist` folder only.